models/Bank.js

```diff
@@ -0,0 +1,38 @@
+import mongoose from "mongoose";
+const { Schema } = mongoose;
+
+const BankSchema = new Schema({
+  userId: {
+    type: String,
+    required: true,
+    unique: true,
+  },
+  bank_name: {
+    type: String,
+    required: true,
+    minLength: 7,
+  },
+  bank_acc: {
+    type: String,
+    required: true,
+    unique: true,
+  },
+  bank_branch: {
+    type: String,
+    required: true,
+  },
+  bank_ifsc: {
+    type: String,
+    required: true,
+    maxLength: 11,
+    minLength: 11,
+  },
+  bank_micr: {
+    type: String,
+    required: true,
+    maxLength: 9,
+    minLength: 9,
+  },
+});
+
+export default mongoose.model("Bank", BankSchema);
```

Comment:
don't export the model object 
do the following instead 
const studentBank = new connector.model("std-bank", studentBankSchema);